Cargo.toml

```diff
@@ -10,8 +10,7 @@ resolver = "2"
 
 members = [
 	"relay/paseo",
-	"relay/paseo/constants",
-	"system-parachains/asset-hub-paseo"
```

Comment:
I think we don't want to remove the asset-hub package.

Reply:
yes for now, that code from asset hub is old and the compilation fails because of incompatibilities. I didn't remove the code to not have a huge diff. We can dismiss this and when we onboard the asset hub we put it back.